sky/serve/service_spec.py

```diff
@@ -7,6 +7,7 @@
 import yaml
 
 from sky.serve import constants
+from sky.serve.load_balancing_policies import LB_POLICIES
```

Comment:
nit: import modules instead of functions/classes.
https://google.github.io/styleguide/pyguide.html#224-decision

Reply:
Also, lets expose this to `sky/serve/__init__.py`?

https://github.com/skypilot-org/skypilot/blob/63e96f49a94a3bdd0263baf52de5eb746b4adc77/sky/jobs/__init__.py#L14